README.md

```diff
@@ -20,32 +20,21 @@ cri-tools aims to provide a series of debugging and validation tools for Kubelet
 - Building a new kubelet container runtime based on CRI.
 - Managing pods/containers for CRI-compatible runtimes by end-users, e.g. pods created by crictl may be removed automatically by kubelet because of non-exist on the kube-apiserver.
 
-## Current Status
-
-Version matrix:
-
-| Kubernetes Version | cri-tools Version | cri-tools branch |
-| ------------------ | ----------------- | ---------------- |
-| ≥ 1.16.x           | ≥ 1.16.x          | master           |
-| 1.15.X             | v1.15.0           | release-1.15     |
-| 1.14.X             | v1.14.0           | release-1.14     |
-| 1.13.X             | v1.13.0           | release-1.13     |
-| 1.12.X             | v1.12.0           | release-1.12     |
-| 1.11.X             | v1.11.1           | release-1.11     |
-| 1.10.X             | v1.0.0-beta.2     | release-1.10     |
-| 1.9.X              | v1.0.0-alpha.1    | release-1.9      |
-| 1.8.X              | v0.2              | release-1.8      |
-| 1.7.X              | v0.1              | release-1.7      |
-
-See the [roadmap](docs/roadmap.md) for information about current and future milestones.
-
-## Release Support Matrix
-
-The [releases](https://github.com/kubernetes-sigs/cri-tools/releases) are in tandem
-with Kubernetes releases in general. As referenced in above version matrix,
-since release `1.16`, we maintain `master` branch of this repo. However, for critical
-fixes in previous releases, we would only consider the releases which are still not EOL
-in Kubernetes releases.
+## Compatibility matrix: cri-tools ⬄ Kubernetes
```

Comment:
the new section doesn't cover compatibility matrix. We need to state which versions of runtime is compatible with which version of cri tools.

You can refer to this page: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cri-api/README.md#version-skew-policy-for-cri-api that states some compatibility requirements

Reply:
I moved the matrix back and added current versions to it. I think full compatibility for 1.28 and 1.27 can be only achieved if we stick to the corresponding k8s version, because of CRI feature changes.

Reply:
1.28 of tools still works with k8s 1.27. Even though some features will not work. I agree we can limit the use to ask to get cri tools of AT LEAST the same minor version. But forcing to use exactly the same sounds excessive. wdyt?

Reply:
Okay, added a note that we recommend to use the same minor versions and updated the table to allow 1.27/28 work together.